### PR TITLE
integrate omnipay-v3-bridge

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
as discussed here: https://github.com/Payum/Payum/pull/679

### CODE REVIEW

File: src/Payum/Core/PayumBuilder.php [hunk 1]
```diff
@@ -627,4 +629,26 @@ protected function buildOmnipayGatewayFactories(GatewayFactoryInterface $coreGat
...
+        if (false == class_exists(\Omnipay\Omnipay::class) || false == class_exists(OmnipayV3GatewayFactory::class)) {
+            return $gatewayFactories;
+        }
+
+        $factory = \Omnipay\Omnipay::getFactory();
+
+        $gatewayFactories['omnipay'] = new OmnipayV3GatewayFactory($factory, [], $coreGatewayFactory);
+        $gatewayFactories['omnipay_direct'] = new OmnipayV3GatewayFactory($factory, [], $coreGatewayFactory);
+        $gatewayFactories['omnipay_offsite'] = new OmnipayV3GatewayFactory($factory, [], $coreGatewayFactory);
+
```
Comment (makasim) [inline src/Payum/Core/PayumBuilder.php]: Could you please remove omnipay_direct and omnipay_offsite

**Comment by dpfaffenbauer** [comment]
removed omnipay-direct and omnipay-offsite.

**Comment by makasim** [comment]
@dpfaffenbauer do you have a twitter account?

**Comment by dpfaffenbauer** [comment]
I do: https://twitter.com/dpfaffenbauer, but I am not really active :D

**Comment by makasim** [comment]
Available since 1.4.3 core, the bridge in its current state was tagged as 1.0.0-alpha1

**Pull request merged**